src/main/java/uk/gov/hmcts/divorce/citizen/notification/SaveAndSignOutNotificationHandler.java

```diff
@@ -27,7 +27,7 @@ public void notifyApplicant(CaseData caseData, UserDetails user) {
         notificationService.sendEmail(
             user.getEmail(),
             SAVE_SIGN_OUT,
-            commonContent.templateVarsForApplicant(caseData, self, partner),
+            commonContent.mainTemplateVars(caseData, null, self, partner),
```

Comment:
why null? cant you get case id from either caseData or user?

Reply:
it's null because it's not used, this template doesn't use the caseId